src/_data/open-source.json

```diff
@@ -27,6 +27,116 @@
                 ]
             }
         ]
-    }
+    }, 
+        "projects":{
+            "projectName":"GitHub projects",
+            "project": [
+                {
+                  "projectName":"My Cv V2 Junior",
```

Comment:
skillName" : "HTML",
                         "percentage" : "100",
                         "color" : "#F0E218"


Add some space for a better readability